Tests/Packet++Test/Tests/HttpTests.cpp

```diff
@@ -300,6 +300,18 @@ PTF_TEST_CASE(HttpResponseParseStatusCodeTest)
 	PTF_ASSERT_EQUAL(pcpp::HttpResponseFirstLine::parseStatusCode(std::string("HTTP/x.y 477").c_str(), 12), pcpp::HttpResponseStatusCode::HttpStatus4xxCodeUnknown, enum);
 	PTF_ASSERT_EQUAL(pcpp::HttpResponseFirstLine::parseStatusCode(std::string("HTTP/x.y 577").c_str(), 12), pcpp::HttpResponseStatusCode::HttpStatus5xxCodeUnknown, enum);
 	PTF_ASSERT_EQUAL(pcpp::HttpResponseFirstLine::parseStatusCode(std::string("HTTP/x.y 600").c_str(), 12), pcpp::HttpResponseStatusCode::HttpStatusCodeUnknown, enum);
+
+
+	// test getMessage()
+	PTF_ASSERT_EQUAL(pcpp::HttpResponseFirstLine::parseStatusCode(std::string("HTTP/x.y 200").c_str(), 12).getMessage(), "OK");
+	PTF_ASSERT_EQUAL(pcpp::HttpResponseFirstLine::parseStatusCode(std::string("HTTP/x.y 404").c_str(), 12).getMessage(), "Not Found");
+
+	std::string testLine;
+	testLine = "HTTP/x.y 404 My Not Found\r\n";
```

Comment:
we can also add a test without `\r`, for example: `"HTTP/x.y 404 My Not Found\n"`

Reply:
done